### ISSUE TITLE
counterfactual-fairness.md's claimed 35.7% violation rate doesn't match what the seeded code produces (and the racial-disparity direction is backwards)

### ISSUE THREAD

**Issue by yakew7** [yakew7/Fair-Code#654]
**Where:** `explainers/counterfactual-fairness.md`, lines 89-198 (the synthetic lending code block + "Expected output" comment at lines 191-196, prose claim at line 199).

**The gap:** the article's own Python code block (synthetic lending example, `np.random.seed(42)`, `n=3000`, `RandomForestClassifier(n_estimators=100, random_state=42)`, `train_test_split(..., random_state=42)`, then `np.random.seed(0)` before the counterfactual re-derivation) claims this "Expected output":
```
Total test records:               600
Counterfactual violations:        214
Violation rate:                   35.7%
  White applicants whose decision flips: 87/356 (24.4%)
  Black applicants whose decision flips: 127/244 (52.0%)
```
Running the exact code as written (verified on the pinned `requirements-lock.txt` versions: numpy 2.4.6, pandas 3.0.3, scikit-learn 1.9.0), deterministically and repeatably, produces:
```
Total test records:                600
Counterfactual violations:         189
Violation rate:                   31.5%
  White applicants whose decision flips: 114/355 (32.1%)
  Black applicants whose decision flips: 75/245 (30.6%)
```
This isn't just an off-by-a-few-percent mismatch - the article's rhetorical point (line 199: "more than one in three applicants... Black defendants flip at 52% vs White at 24.4%, this is the operational signature of race-based decisions") depends on Black applicants flipping much more often than White applicants. The actual reproduction shows the two groups flip at nearly identical rates (32.1% vs 30.6%), which undercuts the example's own argument.

**Repro:** run the full code block verbatim from repo root; deterministic, same result on repeated runs (189/600, 31.5%, White 114/355 32.1%, Black 75/245 30.6%).

**Fix direction:** re-run the code block and paste the actual output (189/600, 31.5%, White 114/355 32.1%, Black 75/245 30.6%), and rewrite the surrounding prose (line 199 and the "When They Diverge" framing) since the current narrative claim about which group is harmed more is not what this synthetic example actually shows once run.
